api/swagger-spec/v1beta1.json

```diff
@@ -2148,15 +2089,15 @@
     "properties": {
      "name": {
       "type": "string",
-      "description": "Name must be unique within a namespace. Is required when creating resources, although some resources may allow a client to request the generation of an appropriate name automatically. Name is primarily intended for creation idempotence and configuration definition. Cannot be updated. More info: http://releases.k8s.io/HEAD/docs/user-guide/identifiers.md#names"
+      "description": "Name must be unique within a namespace. Is required when creating resources, although some resources may allow a client to request the generation of an appropriate name automatically. Name is primarily intended for creation idempotence and configuration definition. Cannot be updated. More info: http://releases.k8s.io/v1.1.0/docs/user-guide/identifiers.md#names"
```

Comment:
Yes I see its the same in release-1.0 as well.
HEAD was replaced by v1.0.0 everywhere in the spec when we cut 1.0.